internal/v1/server.go

```diff
@@ -181,6 +181,9 @@ type binder struct{}
 
 func (b binder) Bind(i interface{}, ctx echo.Context) error {
 	request := ctx.Request()
+	if request.ContentLength == 0 {
+		return nil
+	}
```

Comment:
This now allows creation of endpoints without any body. Previously JSON was enforced, I believe this is okay.

Reply:
Yeah, I think we just didn't need it anywhere, but it was actually a "bug" IMO :)